src/ImGui.NET.SampleProgram.XNA/ImGui.NET.SampleProgram.XNA.csproj

```diff
@@ -5,7 +5,7 @@
         <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
         <AssemblyName>ImGuiNET.SampleProgram.XNA</AssemblyName>
         <RootNamespace>ImGuiNET.SampleProgram.XNA</RootNamespace>
-        <TargetFramework>net472</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
```

Comment:
That's in preview still. Not released until Nov.

Reply:
Yes, but it will be required for running on Apple Silicon since minimum .NET for it will be 6.0.

Reply:
Already out, how about merge? 👯